timeline/templates/timeline/_list_item.html

```diff
@@ -31,6 +31,16 @@
     {% endif %}
   </td>
   <td>
+    {% if event.project and user.is_superuser and site_admin %}
+      <span class="badge badge-info mr-1" title="{{ event.project }}" data-toggle="tooltip" data-placement="top">
+        <i class="iconify"
+           data-icon="{% if event.project.type == 'PROJECT' %}mdi:cube{% else %}mdi:rhombus-split{% endif %}">
+        </i>
+        <a href="{% url 'projectroles:detail' project=event.project.sodar_uuid %}">
```

Comment:
The `a` tag requires the `text-white` class here, otherwise we get a blue link on blue background.